osgp/platform/osgp-adapter-kafka-logging/src/main/resources/osgp-adapter-kafka-logging.properties

```diff
@@ -20,16 +20,16 @@ peakshaving.kafka.consumer.auto.offset.reset=latest
 # KAFKA CONFIG GRID MEASUREMENT PUBLISHED EVENT LOGGING CONSUMER
 # ==================================================================
 
-grid.measurement.published.event.kafka.logging.enabled=false
+scada.measurement.published.event.kafka.logging.enabled=true
```

Comment:
Why was this changed from false to true?

Reply:
to enable logging, changed it back to false to disable logging at the build server